.vib/magento/goss/goss.yaml

```diff
@@ -0,0 +1,30 @@
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
+    stdout: []
+    stderr: []
+  which-mariadb:
+    exec: which mariadb
+    exit-status: 0
+    stdout:
+    - "/opt/bitnami/mysql/bin/mariadb"
+    stderr: []
+    timeout: 0 
+  apache-running:
+    exec: ps aux | grep apache
+    exit-status: 0
+    stdout:
+      - /opt/bitnami/apache/bin/httpd
+    stderr: []
+    timeout: 0
+file:    
+  /opt/bitnami/magento:
+    filetype: symlink
+    linked-to: /bitnami/magento
+    exists: true
+  /opt/bitnami/magento/package.json.sample:
+    filetype: file
+    exists: true
+    contains: 
+      - "Magento Commerce Inc."
```

Comment:
I don't think it is worth checking this file, it is just a sample.

I might be wrong since it's been ages since the last time I checked magento, but I think there was a configuration file somewhere. Maybe it is worth checking it instead.

Reply:
I've added a test that checks a real config file instead of checking this one. 
WDYT about the new test?

Reply:
I like this one now 😁 Thanks!